### PR TITLE
API-12935 Added ads_traffic greeting description

### PR DESCRIPTION
**Pull request by dmasior**
# Deploy preview

Scroll down to the checks section for the link to the deploy preview of your branch.

# Links

Link your Jira ticket.

- [Jira](https://livechatinc.atlassian.net/browse/API-12815)

# Description
New greeting trigger `ads_traffic`

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

File: src/pages/management/configuration-api/v2.0/index.mdx [hunk 1]
```diff
@@ -2384,6 +2384,7 @@ And here's a list of the greeting rules that you can use:
 | `search_keyword`    | When a visitor has searched for a specific keyword                                                                  |
 | `custom_variable`   | The greeting will be sent when the specified variable is met [documented below](#custom-variable)                   |
 | `url_funnel`        | The visitors who visit the specified sequence of websites will receive the greeting [documented below](#url-funnel) |
+| `ads_traffic`       | The greeting will be sent when customer fist visited page contains ad provider parameters                           |
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v2.0/index.mdx:2387]: I am not sure what we want to achieve here🤔 It is not clear to me when I read it. Do we want?
1. The greeting will be sent when the customer's first visited page contains ad provider parameters
or
2. The greeting will be sent when the customer first visits the page that contains ad provider parameters
I assume we want option 1, but I prefer to confirm😜

File: src/pages/management/configuration-api/v2.0/index.mdx [hunk 2]
```diff
@@ -2562,6 +2563,119 @@ rules[0][operator]=contains"
 
 <Text>
 
+### Ads traffic
+
+The greeting will be sent when the customer first visit url contains specified ads parameters.
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v2.0/index.mdx:2568]: Here the same case as above
1. The greeting will be sent when the customer's first visited URL contains specified ads parameters.
2. The greeting will be sent when the customer first visits the URL containing specified ads parameters.

File: src/pages/management/configuration-api/v2.0/index.mdx [hunk 3]
```diff
@@ -2562,6 +2563,119 @@ rules[0][operator]=contains"
...
+
+When using the `ads_traffic` greeting rule, you also may provide `variable_name` and `variable_value`.
+For example, providing `variable_name` as `facebook` and `variable_value` as `1234` will send the greeting when
+the customer first visit url contains `fbclid=1234`.
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v2.0/index.mdx:2572]: here I assume we mean `...the customer first visits URL containing...`

File: src/pages/management/configuration-api/v2.0/index.mdx [hunk 4]
```diff
@@ -2562,6 +2563,119 @@ rules[0][operator]=contains"
...
+You also can add the optional `operator` parameter (defaults to `equals`).
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v2.0/index.mdx:2573]:
```suggestion
You can also add the optional `operator` parameter (defaults to `equals`).
```

File: src/pages/management/configuration-api/v2.0/index.mdx [hunk 5]
```diff
@@ -2562,6 +2563,119 @@ rules[0][operator]=contains"
...
+<ul>
+  <li><code>facebook</code> for <code>fbclid</code>,</li>
+  <li><code>google</code> for <code>dclid</code>, <code>gbraid</code>, <code>gclid</code>, <code>gclsrc</code>, <code>
+  wbraid</code>,</li>
+  <li><code>microsoft</code> for <code>msclkid</code>,</li>
+  <li><code>twitter</code> for <code>twclid</code>,</li>
+  <li><code>yahoo</code> for <code>yclid</code>,</li>
+</ul>
+
+Max allowed `variable_value` length is 255 characters.
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v2.0/index.mdx:2585]:
```suggestion
The maximum allowed `variable_value` length is 255 characters.
```

File: src/pages/management/configuration-api/v2.0/index.mdx [hunk 6]
```diff
@@ -2562,6 +2563,119 @@ rules[0][operator]=contains"
...
+
+When not providing `variable_name` and `variable_value`, the greeting will be sent when the customer first visit url
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v2.0/index.mdx:2587]: Like above, I believe we want
When not providing `variable_name` and `variable_value`, the greeting will be sent when the customer first visits the URL containing any of the ads parameters.

File: src/pages/management/configuration-api/v2.0/index.mdx [hunk 7]
```diff
@@ -2562,6 +2563,119 @@ rules[0][operator]=contains"
...
+contains any of the ads parameters.
+
+When providing only `variable_name`, the greeting will be sent when the customer first visit url containing any of the
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v2.0/index.mdx:2590]: Like above, I belive we want
When providing only `variable_value`, the greeting will be sent when the customer first visits the URL containing any of the ads parameters with the specified values. <- I belive we should make value plural

File: src/pages/management/configuration-api/v2.0/index.mdx [hunk 8]
```diff
@@ -2562,6 +2563,119 @@ rules[0][operator]=contains"
...
+ads parameters coming from the provider with the specified name.
+
+When providing only `variable_value`, the greeting will be sent when the customer first visit url contains any of the
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v2.0/index.mdx:2593]: Like above, I believe we want
When providing only variable_value, the greeting will be sent when the customer first visits the URL containing any of the ads parameters with the specified values. <- I believe we should make value plural

File: src/pages/management/configuration-api/v2.0/index.mdx (hunk 1 above)
Comment (dmasior) [reply src/pages/management/configuration-api/v2.0/index.mdx]: Option 2 seems fine

File: src/pages/management/configuration-api/v2.0/index.mdx [hunk 9]
```diff
@@ -2562,6 +2563,120 @@ rules[0][operator]=contains"
...
+  <li><code>twitter</code> for <code>twclid</code>,</li>
+  <li><code>yahoo</code> for <code>yclid</code>,</li>
+</ul>
+
+The maximum allowed `variable_value` length is 255 characters.
+
+When not providing `variable_name` and `variable_value`, the greeting will be sent when the customer first visits the
+URL containing any of the ads parameters.
+
+When providing only `variable_value`, the greeting will be sent when the customer first visits the URL containing any of
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v2.0/index.mdx:2591]: Shouldn't it be `variable_name` here?🤔 Except this one, everything looks good :)

**Review by kamilatokarska — APPROVED** [review]
:)

**Pull request merged**